core/src/main/java/org/apache/gravitino/listener/api/event/TagFailureEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.gravitino.listener.api.event;
+
+public class TagFailureEvent extends FailureEvent {
+    public TagFailureEvent(String user, Exception exception) {
+        super(user, null, exception);
```

Comment:
identifier should not be null for tag event.

Reply:
What should I pass here?